design/sketch.webidl

```diff
@@ -580,8 +580,11 @@ interface WebGPUFence {
 
 // Queue
 interface WebGPUQueue {
+    readonly attribute WebGPUQueueFamily family;
+
     void submit(sequence<WebGPUCommandBuffer> buffers);
     WebGPUFence insertFence();
+    void wait(WebGPUFence);
```

Comment:
Queues can wait but they can't signal?

Reply:
I think this commit is missing a rebase.